conf/scalardb-graphql-custom-values.yaml

```
@@ -0,0 +1,70 @@
+replicaCount: 3
+
+scalarDbGraphQlConfiguration:
+  #
+  # DynamoDB
```

I think it looks like it's usable only for DynamoDB. (understand you took it from the existing custom confg.)

```suggestion
  # Specify the configuration of a database you use.
  # The following is the example configuration of DynamoDB.
```

---

Ping @ymorimo

---

@brfrn169 Thank you for notifying me!